client/document/browsehappy.jsx

```diff
@@ -10,7 +10,17 @@ import Head from 'calypso/components/head';
 import EmptyContent from 'calypso/components/empty-content';
```

Comment:
It looks like no translation was being done on this page before, but now that it's going to be shown to more people it's probably important to make sure it's translated.